papers/ice_nucleation_2024/calibration.jl

```diff
@@ -167,7 +264,7 @@ function calibrate_J_parameters(FT, IN_mode, params, IC, y_truth, Γ,; perfect_m
 
     prior = create_prior(FT, IN_mode, perfect_model = perfect_model)
     N_ensemble = 10      # runs N_ensemble trials per iteration
-    N_iterations = 15    # number of iterations the inverse problem goes through
+    N_iterations = 150    # number of iterations the inverse problem goes through
```

Comment:
Is this really needed? 150?

Reply:
Think I needed it before when the calibrations weren't working right and things didn't converge early enough. I can change it to 75 but if I set it to anything lower, one of the experiments doesn't seem to really converge (IN05_19)